### DISCUSSION TITLE
Official example const myAction = new Action('com.example-url.plugin-test.sdPlugin.action'); Report an error

### DISCUSSION THREAD

**Discussion by Qingpo-o** [elgatosf/elgato#119] [Help] [answered]
In the plug-in example of javasprict provided by the official website, I reported an error  "const myAction = new action (' com.example-url.plugin.test.sdplugin.action ') " when running; Instantiation error, only $SD.onConnected(({actionInfo, appInfo, connection, messageType, port, uuid}) => {
console.log('Stream Deck connected! ');
}); This function can print normally, button press and release events can not trigger, can you help me solve

**Comment by ChekTek** [comment, marked as answer]
Hi @Qingpo-o 

You'll need to make sure the `UUID` you provide matches the `UUID` in the manifest.json file. 

We are in the process of migrating away from that version of the plugin template. I would recommend using the new [CLI tool](https://www.npmjs.com/package/@elgato/cli) for new plugins.
